resteasy-wadl/src/main/java/org/jboss/resteasy/wadl/ResteasyWadlServlet.java

```diff
@@ -18,12 +18,19 @@
 
 /**
  * @author <a href="mailto:l.weinan@gmail.com">Weinan Li</a>
+ * This servlet does not support grammars.
+ * Use @org.jboss.resteasy.wadl.ResteasyWadlDefaultServlet instead.
  */
+@Deprecated
```

Comment:
On master we'll likely remove this instead of deprecating (reminder)

Reply:
added TODO comments to remind this